tests/src/main/java/net/neoforged/neoforge/debug/data/DataMapTests.java

```diff
@@ -305,6 +310,43 @@ protected void gather() {
                 .thenSucceed());
     }
 
+    @GameTest
+    @EmptyTemplate
+    @TestHolder(description = "Tests if the data map update event works", groups = EventTests.GROUP)
+    static void dataMapUpdateEventTest(final DynamicTest test, final RegistrationHelper reg) {
+        final DataMapType<Item, Integer> dataMap = reg.registerDataMap(DataMapType.builder(
```

Comment:
do you manually write `final` every time?

Reply:
Yes, it's a reflex :P